src/styles/blocks/page.scss

```diff
@@ -0,0 +1,49 @@
+.page {
+  font-family: "Poppins", sans-serif;
+  background-color: $color-dark-gray;
+  scroll-behavior: smooth;
+  color: #fff;
+
+  &__body--with-menu {
+    overflow: hidden;
+  }
+
+  &__menu {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+
+    transform: translateX(100%);
+    transition: transform 1.5s;
+
```

Comment:
maybe 1.5s is too much?

Reply:
I like how it slowly shows up, but I guess it's more of a personal preference 😅